### PR TITLE
161766223 two up display

### PR DESCRIPTION
**Pull request by mklewandowski**
Add a PopulationSpace component that displays 2 side-by-side panels which will eventually house the population sim on the right and a graph/data view on the left.  Remove the now obsolete InvestigationPanel component.  Display the PopulationSpace component when the user clicks on the population button in the investigation dialog.  Update the UI model to contain population graph state (this will need to be reworked to have greater complexity.  We could use an enum that reflects the current space that is visible.  This is probably best done in its own story and branch).

### CODE REVIEW

File: src/components/population-space.tsx [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  public render() {
+    return (
+      <div className="population-space">
+        {this.RenderLeftPanel()}
+        {this.RenderRightPanel()}
+      </div>
+    );
+  }
+
+  private RenderLeftPanel() {
```
Comment (ekosmin) [inline src/components/population-space.tsx]: Should probably be lowercased

File: src/components/population-space.tsx [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+        <div className="header">
+          <div className="title">Investigate: Population</div>
+        </div>
+        <div className="content">content here</div>
+      </div>
+    );
+  }
+
+  private RenderRightPanel() {
+    const {ui} = this.stores;
```
Comment (ekosmin) [inline src/components/population-space.tsx]: Agreed about eventually refactoring state - we may want to create a separate UI store for each space

File: src/components/population-space.tsx (hunk 1 above)
Comment (mklewandowski) [reply src/components/population-space.tsx]: Yes!  I got too caught up with the uppercase on the exported React component name.  Will change.

File: src/components/population-space.tsx (hunk 2 above)
Comment (mklewandowski) [reply src/components/population-space.tsx]: Agree.  We'll need a story for that.

**Comment by sfentress** [comment]
:+1: Looks good.

**Pull request merged**